recipes/exhale/exhale/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "exhale" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/exhale-{{ version }}.tar.gz
+  sha256: ce936be4330db0d3ea96dbda8144cda202d2e57cd52f50f89c7b0393b528e6c5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - breathe
+    - bs4
+    - lxml
+    - pip
+    - python
+    - six
+    - sphinx >=1.6.1
+  run:
+    - breathe
+    - bs4
+    - lxml
+    - python
+    - six
+    - sphinx >=1.6.1
```

Grayskull gets these deps as

```suggestion
requirements:
  host:
    - pip
    - python >=3.6
  run:
    - beautifulsoup4
    - breathe >=4.32.0
    - docutils >=0.12
    - lxml
    - python >=3.6
    - six
    - sphinx >=3.0,<5
```

---

I also used grayskull to generate this using the following command locally: 
```
grayskull pypi exhale=0.2.3
```